q2_gneiss/citations.bib

```diff
@@ -0,0 +1,10 @@
+@article{morton2017balance,
+  title={Balance trees reveal microbial niche differentiation},
+  author={Morton, James T and Sanders, Jon and Quinn, Robert A and McDonald, Daniel and Gonzalez, Antonio and V{\'a}zquez-Baeza, Yoshiki and Navas-Molina, Jose A and Song, Se Jin and Metcalf, Jessica L and Hyde, Embriette R and others},
```

Comment:
Could you update this to the full author list?

Reply:
👍